atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -87,14 +87,20 @@ public int timeBetweenConnectionEvictionRunsSeconds() {
         return 20;
     }
 
+    /**
+     * The duration we wait to check if the all underlying Cassandra nodes are healthy.
```

Comment:
Think this message could be clearer.  This is the period between refreshing the C* pools.  Worth calling out this is how hosts get whitelisted after being blacklisted.

Do we check the health of all hosts or just those on the blacklist?